examples/select_parser.py

```diff
@@ -1,3 +1,4 @@
+# -*- coding: utf-8 -*-
```

Comment:
This is the default encoding for Py3 now, we won't be adding back any Py2 compatibility code in core or examples.

Reply:
(The emojis *are* a nice touch.)

Reply:
IME by far the best way to test code for unicode-correctness 🦄